concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -101,40 +94,87 @@ def report_computed_task(timestamp = None, subtask_id = None, task_to_compute =
         )
 
 
+def create_task(task_to_compute_timestamp, task_id, compute_task_def_deadline):
+    return task_to_compute(
+        timestamp=task_to_compute_timestamp,
+        compute_task_def=compute_task_def(
+            task_id=task_id,
+            subtask_id=task_id,
+            deadline=compute_task_def_deadline,
+        )
+    )
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
+                               ["concent_messaging_time",
+                                "subtask_verification_time",
+                                "force_acceptance_time",
+                                "token_expiration_time"])
+
+
+def get_protocol_constants(cluster_url):
+    url = f"{cluster_url}/api/v1/protocol-constants/"
+    resp = requests.get(url)
+    json = resp.json()
```

Comment:
We should not try to parse response as JSON unless `Content-Type` says it's JSON. If it's e.g. HTML or text (because something went wrong) it would be better to print it.

Also, this is WIP. We will be using these constants all over the place soon.

Reply:
This is still not fixed.

If the server returns HTML (with `Content-Type` of `text/html`), this code will try to parse it as JSON.